example/lib/sprite_body_sample.dart

```diff
@@ -18,36 +19,36 @@ class Pizza extends SpriteBodyComponent {
   Body createBody() {
     final PolygonShape shape = PolygonShape();
 
-    final v1 = Vector2(0, size.y / 2);
-    final v2 = Vector2(size.x / 2, -size.y / 2);
-    final v3 = Vector2(-size.x / 2, -size.y / 2);
-    final vertices = [v1, v2, v3];
-    shape.set(vertices, vertices.length);
-
-    final fixtureDef = FixtureDef();
-    fixtureDef.userData = this; // To be able to determine object in collision
-    fixtureDef.shape = shape;
-    fixtureDef.restitution = 0.3;
-    fixtureDef.density = 1.0;
-    fixtureDef.friction = 0.2;
-
-    final bodyDef = BodyDef();
-    bodyDef.position = viewport.getScreenToWorld(_position);
-    bodyDef.angle = (_position.x + _position.y) / 2 * 3.14;
-    bodyDef.type = BodyType.DYNAMIC;
+    final vertices = [
+      Vector2(-size.x / 2, -size.y / 2),
```

Comment:
We should add a method on Vector2 extension 'middle', we do that on a lot of places. Just a random thought not needed for this PR